### PR TITLE
Update Known-issues.md to include Symantec DLP

### PR DESCRIPTION
**Pull request by wayvad**
This change adds the recently discovered interop issue with Symantec DLP into the known issue list.

See ICM https://icm.ad.msft.net/imp/v3/incidents/details/128785963/home for more info.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by LolaJacobsen** [comment]
@marclauinger - for your review.

File: Teams/Known-issues.md [hunk 1]
```diff
@@ -340,3 +340,7 @@ This article lists the known issues for Microsoft Teams, by feature area.
 |**Issue title**|**Behavior / Symptom**|**Known workaround**|**Discovery date**|
 |:-----|:-----|:-----|:-----|
 |Unable to move, delete or rename files after editing  <br/> |After a file is edited in Teams it cannot be moved or renamed or deleted immediately <br/> |This is currently a known issue and the workaround is to wait for some time before making administrative changes.  <br/> |03/12/19  <br/> |
+
+|**Issue title**|**Behavior / Symptom**|**Known workaround**|**Discovery date**|
+|:-----|:-----|:-----|:-----|
+|Interoperability issue between Symantec DLP and Teams <br/> |Symantec DLP Endpoint agents can interfere with Teams process, which can then lead to a launch or exit failure.  <br/> |Exclude (whitelist) Teams.exe from the Symantec DLP's Endpoint agents as described in this Symantec's <a href="https://support.symantec.com/us/en/article.TECH220322.html">article</a> . <br/> |07/15/19  <br/> |
```
Comment (nenonix) [inline Teams/Known-issues.md]:
```suggestion
|Interoperability issue between Symantec DLP and Teams <br/> |Symantec DLP Endpoint agents can interfere with Teams process, which can then lead to a launch or exit failure.  <br/> |Exclude (whitelist) Teams.exe from the Symantec DLP's Endpoint agents as described in this <a href="https://support.symantec.com/us/en/article.TECH220322.html">Symantec support article</a> . <br/> |07/15/19  <br/> |
```

Comment (wayvad) [reply Teams/Known-issues.md]: Thanks. Fixed.

**Comment by scanum** [comment]
@LolaJacobsen PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**